.github/workflows/linux_ubuntu.yml

```diff
@@ -171,12 +166,8 @@ jobs:
           path: ${{ github.workspace }}/release/svalinn-plugin_${{ matrix.os }}-${{ matrix.os_version }}_cubit_${{ matrix.cubit }}.tgz
           if-no-files-found: error # 'warn' or 'ignore' are also available, defaults to `warn` 
     
-      - if: github.event_name == 'release'
-        name: Upload binaries into the release
-        uses: svenstaro/upload-release-action@v2
+      - name: Upload binaries into the release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
```

Comment:
Is there a substantive reason to change the condition?  Why not put the conditional first?

Reply:
That is just how the example suggested to do it.
https://github.com/softprops/action-gh-release